### DISCUSSION TITLE
Clock Drift - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#797] [Showcases]
Discuss the "Clock Drift" showcase here. Ideas and recommendations are welcome.

**Comment by windy32** [comment]
With OMNet++ 6.0.1 / INET 4.4.1, it seems that no vector data of "clock drift" is generated.

**Reply by windy32** [reply]
> With OMNet++ 6.0.1 / INET 4.4.1, it seems that no vector data of "clock drift" is generated.

It is mentioned in the next chapter "Using gPTP" that related statistic is "timeDifference:vector".

However in most cases, users would read "Clock Drift" first before "Using gPTP". I suggest that a similar note be added to the "Clock Drift" section.

**Reply by adamgeorge309** [reply]
Added in master. Thanks for the suggestion.

**Comment by Jannusch** [comment]
Hi, minor thing: The link that should point to the example files in the git repo is broken. Between "inet-framework" and "tree" the word "inet" is missing.
